packages/blade/src/_helpers/testing/renderWithTheme.native.tsx

```diff
@@ -1,8 +1,8 @@
 import type { RenderAPI } from '@testing-library/react-native';
 import { render } from '@testing-library/react-native';
 import type { ReactElement } from 'react';
-import { BladeProvider } from '../../components/BladeProvider';
-import { paymentTheme } from '../../tokens/theme';
+import { BladeProvider } from '~components/BladeProvider';
```

Comment:
Does this happen automatically now? Or you did these manually?

Reply:
I manually replaced all the old imports 😢

Reply:
Won't `eslint --fix` do this automatically 🤔

Reply:
Nope since even when you add path aliases. 

`../../../../../../Button` is still valid import.

Reply:
but does vscode autoimport imports it as alias or still with `../../` things?

Reply:
Aliased paths. 

https://user-images.githubusercontent.com/35374649/180220757-3003eb21-44e0-4aeb-8597-7629919502ec.mov

Reply:
just curious: why haven't you enabled format on auto-save? doesn't it feels weird to open the command palette and fix eslint errors always? 🤔

Reply:
😆 I like to have some control over my code. I find format on save annoying while in active development.